packages/react-components/react-motion/library/src/slots/presenceMotionSlot.tsx

```diff
@@ -0,0 +1,79 @@
+import {
+  SLOT_ELEMENT_TYPE_SYMBOL,
+  SLOT_RENDER_FUNCTION_SYMBOL,
+  type SlotComponentType,
+  type SlotRenderFunction,
+} from '@fluentui/react-utilities';
+import * as React from 'react';
+
+import type { PresenceComponentProps } from '../factories/createPresenceComponent';
+import type { MotionParam } from '../types';
+
+type PresenceMotionSlotReturnProps<MotionParams extends Record<string, MotionParam> = {}> = Pick<
```

Comment:
@bsunderhus to limit/reduce `props` usable with object syntax, currently it's `'imperativeRef' | 'onMotionFinish' | 'onMotionStart'`. For more complex usages, `children` function should be used where all interface will be available.

I am hesitant to expose more as might break components, for example, `unmountOnExit: false` in the case of `Tree` might cause infinite loops [on this case](https://react.fluentui.dev/?path=/docs/components-tree--default#inline-styling-tree-item-level)).

I marked `PresenceMotionSlotReturnProps` as `@internal` to ensure that it will not leak.